docs/how-to-guides.md

```diff
@@ -54,15 +54,15 @@ You can change the `RESOURCE_NAME_PREFIX` and `ENVIRONMENT_TAG` attribute in the
 To add an additional service to this template, you need to ensure the following:
 
 1. Add the required Bicep definition for the Azure resource to host the service in the [infra folder](../deploy/infra/).
-1. Add a reference to the module in the [main.bicep file](../deploy/infra/main.bicep). To learn more about Bicep [take a look here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/bicep-overview).
-1. Add the service code.
-1. To build your service as part of the [build-release workflow](../.github/workflows/build_release.yaml), add the required build job at the beginning of the workflow.
-1. For any additional actions required as part of deploying the service, you can add the required steps to the [install.sh script](../deploy/scripts/install.sh).
+2. Add a reference to the module in the [main.bicep file](../deploy/infra/main.bicep). To learn more about Bicep [take a look here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/bicep-overview).
+3. Add the service code.
+4. To build your service as part of the [build-release workflow](../.github/workflows/build_release.yaml), add the required build job at the beginning of the workflow.
+5. For any additional actions required as part of deploying the service, you can add the required steps to the [install.sh script](../deploy/scripts/install.sh).
```

Comment:
Is the only change here to increment the numbers?  If so, that's usually not needed as Markdown will auto-increment if you use 1. for all of them.

Were you seeing it not auto-increment somewhere (other than pure markdown)